src/components/OrgPostCard/OrgPostCard.tsx

```
@@ -488,8 +1054,11 @@
                 variant="primary"
                 type="submit"
                 data-testid="update-post-submit"
+                disabled={postFormState.attachments.some((a) => a.isUploading)}
               >
-                Save
+                {postFormState.attachments.some((a) => a.isUploading)
+                  ? tCommon('uploading') + '...'
+                  : tCommon('save')}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)